pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
```

Comment:
Why not use pytest fixtures? I don't think this is bad practice but I've already been there and then refactored it as pytest.fixtures and the code got much cleaner :)